google-ads-examples/src/main/java/com/google/ads/googleads/examples/remarketing/UploadConversionAdjustment.java

```diff
@@ -157,26 +154,29 @@ private void runExample(
       GoogleAdsClient googleAdsClient,
       long customerId,
       long conversionActionId,
-      String gclid,
+      String orderId,
       String adjustmentType,
-      String conversionDateTime,
       String adjustmentDateTime,
-      @Nullable Float restatementValue) {
+      @Nullable Float restatementValue)
+      throws InvalidProtocolBufferException {
     // Gets the conversion adjustment enum value from the adjustmentType String.
     ConversionAdjustmentType conversionAdjustmentType =
         ConversionAdjustmentType.valueOf(adjustmentType);
 
-    // Associates conversion adjustments with the existing conversion action.
-    // The GCLID should have been uploaded before with a conversion.
+    // Applies the conversion adjustment to the existing conversion.
     ConversionAdjustment conversionAdjustment =
         ConversionAdjustment.newBuilder()
             .setConversionAction(ResourceNames.conversionAction(customerId, conversionActionId))
             .setAdjustmentType(conversionAdjustmentType)
-            .setGclidDateTimePair(
-                GclidDateTimePair.newBuilder()
-                    .setGclid(gclid)
-                    .setConversionDateTime(conversionDateTime)
-                    .build())
+            // Sets the orderId to identify the conversion to adjust.
+            .setOrderId(orderId)
+            // As an alternative to setting orderId, you can provide a GclidDateTimePair, but
```

Comment:
Docs state that `gclid_date_time_pair is optional but may be set in addition to the order_id.` so this comment should be rephrased to reflect that. I'm not sure what the best-case would be, but I imagine the recommendation would be to provide both gclid date time _and_ an order ID. Josh perhaps you know better than I do on that.

Reply:
Just realized that we want to steer users towards order ID and away from gclid date/time pairs. So I think the comment should note that both can be set, but that we recommend always setting order ID.

Reply:
I confirmed separately that we should not recommend passing both `order_id` and `gclid_date_time_pair`, so I'll leave this section as-is.